commons/passive/garmin/garmin_generic_device_info.avsc

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "GarminGenericDeviceInfo",
+  "type": "record",
+  "namespace": "org.radarcns.passive.garmin",
+  "doc":"Device information for Garmin devices.",
+  "fields": [
+
+   {
+              "name":"time",
+              "type":"double",
+              "doc":"Last updated time."
+           },
+           {
+              "name":"timeReceived",
+              "type":"double",
+              "doc":"Device receiver timestamp in UTC (s)."
+           },
+
+    {
+      "name": "macAddress",
+      "type": "string",
+       "doc":"MAC address as received from Garmin device."
```

Comment:
I don't think we should be collecting `macAddress`. It can be put in the ManagementPortal source properties (see https://github.com/RADAR-base/radar-commons-android/blob/0d6a16e1a2b931bba038828ceb836d466c189be1/plugins/radar-android-empatica/src/main/java/org/radarbase/passive/empatica/E4Manager.kt#L133), but it should not be put in the main dataset to avoid identifiable data there.

Reply:
Will be removed from latest version.